eng/Versions.props

```diff
@@ -1,16 +1,16 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <!-- MSTest version -->
-    <VersionPrefix>3.7.0</VersionPrefix>
```

Comment:
Same for this, this also needs to be reverted, I do this because locally we produce 3.7.0-dev version but then we build on server and produce 3.7.0-previewXXXX version. `preview` is further in alphabetical order than `dev` so the server versions are newer locally. And so when I try to consume the locally built packages in TA or test project nuget warns about package downgrades and uses the preview instead.